jetty-websocket/jetty-websocket-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -281,13 +281,14 @@ public long getConnectTimeout()
     }
 
     /**
-     * Set the timeout for connecting to the remote server.
+     * Set the timeout for establishing a WebSocket connection with the remote server.
      *
      * @param ms the timeout in milliseconds
      */
     public void setConnectTimeout(long ms)
     {
         getHttpClient().setConnectTimeout(ms);
+        getHttpClient().setIdleTimeout(ms);
```

Comment:
Not really sure this is a good idea. We do already have a `WebSocketClient.setIdleTimeout()` but sets the idleTimeout on the websocket configuration and is not a shortcut for the HttpClient setter.

Reply:
It's not a good idea.